team-agreements/definition-of-done/readme.md

```diff
@@ -1,19 +1,33 @@
-# Team Definition of Done Agreement
+# Definition of Done
 
-## Content TODO's
+To close a user story, a sprint, or a milestone it is important to verify that the tasks are complete.
 
-- Solidify recommended checklist as opposed to providing example (see [cobalt](https://github.com/microsoft/cobalt/wiki/Definition-of-Done) definition of done)
+The development team should decide together what their definition of done is and document this in the project. Below are some examples of checks to verify that the user story, sprint, task is completed.
 
-Software Engineering is not just implementing functionality and writing code, but also other associated tasks and activities to ensure all requirements are fulfilled when you finished your work. This is closely relevant to `done-done` term, which stands for a simple list of activities like writing code/unit tests/release notes/design docs and so on that adds verifiable/demonstrable value to the product.
+## Feature/User Story
 
-Below is an example Done-Done checklist you should follow to ensure everything is done (may vary depending on project/client):
+- [] Acceptance criteria are met
+- [] Refactoring is complete
+- [] Code builds with no error
+- [] Unit tests are written and pass
+- [] Existing Unit Tests pass
+- [] Sufficient diagnostics/telemetry are logged
+- [] Code review is complete
+- [] UX review is complete (if applicable)
+- [] Documentation is updated
+- [] The feature is merged into the develop branch
+- [] The feature is signed off by the product owner
 
-- Is covered by unit and integration tests YES/NO
-- Is continuously tested in each environment YES/NO
-- Does the code coverage meet the minimum threshold YES/NO
-- Is sufficient diagnostic/user telemetry and dashboarding YES/NO
-- Is sufficient public documentation YES/NO
-- Feature owner sign off YES/NO
-- [if applicable] Customer value demo screencast is recorded and sent to the team
-YES/NO
-- [if applicable] Is Relevant architectural review is done YES/NO
+## Sprint Goal
+
+- [] Definition of Done for all user stories included in the sprint are met

```

Comment:
nit: Definition of Done -> casing for 'Done' is different than on line 5. Should we make it consistent?